AABB_tree/include/CGAL/AABB_polyhedron_triangle_primitive.h

```diff
@@ -154,8 +155,8 @@ namespace CGAL {
     };  // end class AABB_polyhedron_triangle_primitive
 
 
-
 }  // end namespace CGAL
+/// \endcond
```

Comment:
Except if I'm wrong, our rules says that after two releases, we can remove deprecated classes. Theses classes were deprecated since 4.3